Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -228,6 +228,14 @@ public static GoogleCredential FromAccessToken(string accessToken, IAccessMethod
         /// by calling <see cref="CreateScoped(IEnumerable{string})"/>
         /// </description>
         /// </item>
+        /// <item>
+        /// <description>
+        /// <see cref="ImpersonatedCredential"/> is not scoped by default but when used without
```

Comment:
If the original credential is explicitly scoped, does that have any bearing on whether the impersonated credential is scoped? I think it would be good to mention that here (either way).